Makefile

```diff
@@ -154,6 +154,26 @@ test-conformance: $(BIN)/conformance_test_runner $(BUILD)/protobuf-conformance
 		--text_format_failure_list packages/protobuf-conformance/conformance_failing_tests_text_format.txt \
 		packages/protobuf-conformance/bin/conformance_esm.js
 
+TS_VERSIONS = 4.1.2 \
+			  4.2.4 \
+			  4.3.5 \
+			  4.4.4 \
+			  4.5.2 \
+			  4.6.4 \
+			  4.8.1-rc
```

Comment:
Can you move those to a single line, and add the line at the end of the other variable declarations at the top of the file?